.github/workflows/post_benchmark_comment.yaml

```diff
@@ -0,0 +1,106 @@
+      should-run: ${{ steps.configure.outputs.should-run }}
+      runner-env: ${{ steps.configure.outputs.runner-env }}
+      runner-group: ${{ steps.configure.outputs.runner-group }}
+    steps:
+      - name: "Checking out repository"
+        uses: actions/checkout@e2f20e631ae6d7dd3b768f56a5d2af784dd54791 # v2.5.0
+        with:
+          # We need the parent commit to do a diff
+          fetch-depth: 2
+      - name: "Configuring CI options"
+        id: configure
+        run: |
+          # Just informative logging. There should only be two commits in the
+          # history here, but limiting the depth helps when copying from a local
+          # repo instead of using checkout, e.g. with
+          # https://github.com/nektos/act where there will be more.
+          git log --oneline --graph --max-count=3
+          ./build_tools/github_actions/configure_ci.py
+
+  check_benchmark_job:
+    needs: setup
+    if: needs.setup.outputs.should-run == 'true'
+    runs-on: ubuntu-20.04
+    outputs:
+      job-conclusion: ${{ steps.check.outputs.job-conclusion }}
+    steps:
+      - name: "Checking benchmark processing job"
+        id: check
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          BENCHMARK_PROCESS_JOB: process_benchmark_results
+        run: |
+          gh api \
+            -H "Accept: application/vnd.github+json" \
+            --jq ".jobs | map(select(.name==\"${BENCHMARK_PROCESS_JOB}\"))[0] | .conclusion" \
+            "/repos/${GITHUB_REPOSITORY}/actions/runs/${PR_CI_RUN_ID}/attempts/${PR_CI_RUN_ATTEMPT}/jobs" \
+            > conclusion
+          echo "job-conclusion=$(cat conclusion)" >> "${GITHUB_OUTPUT}"
+
+  post_benchmark_comment:
+    needs: [setup, check_benchmark_job]
+    if: |
+      needs.setup.outputs.should-run == 'true' &&
+      needs.check_benchmark_job.outputs.job-conclusion == 'success'
+    runs-on: ubuntu-20.04
+    steps:
+      - name: "Checking out repository"
+        uses: actions/checkout@7884fcad6b5d53d10323aee724dc68d8b9096a2e # v2
+      - name: "Downloading artifacts"
+        id: download
+        env:
+          # Currently GitHub managed runner doesn't have GCP credential. Fetch
+          # from public URL directly.
+          PR_CI_GCS_URL: https://storage.googleapis.com/iree-github-actions-${{ env.PR_CI_STAGE }}-artifacts/${{ env.PR_CI_RUN_ID }}/${{ env.PR_CI_RUN_ATTEMPT }}
+          BENCHMARK_COMMENT_ARTIFACT: benchmark-comment.json
+        run: |
+          wget -O "${BENCHMARK_COMMENT_ARTIFACT}" \
+            "${PR_CI_GCS_URL}/${BENCHMARK_COMMENT_ARTIFACT}"
+          echo "benchmark-comment-artifact=${BENCHMARK_COMMENT_ARTIFACT}" >> "${GITHUB_OUTPUT}"
+      - name: Posting comments
+        # TODO(#11076): Temporarily disabled until we migrate the database.
+        if: false
+        env:
+          GITHUB_TOKEN: ${{ secrets.WRITE_ACCESS_TOKEN }}
+          GIST_BOT_USER: github-actions-bot
+          GIST_BOT_TOKEN: ${{ secrets.GIST_BOT_TOKEN }}
```

Comment:
Follow-up task reminder: Add the gist bot token to the secrets

Reply:
Done